maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,735 @@
+    """End the global session trace."""
+    global _session_trace
+    
+    if _session_trace:
+        _session_trace.end()
+        maxim_logger.flush()
+        _session_trace = None
+        _global_maxim_trace.set(None)
+        scribe().debug("[MaximSDK] Ended session trace")
+
+
+def instrument_pydantic_ai(maxim_logger: Logger, debug: bool = False):
```

Ruff (0.12.2)

`instrument_pydantic_ai` is too complex (63 > 10)

(C901)